packages/ag-charts-community/src/scene/layersManager.ts

```diff
@@ -52,6 +52,9 @@ export class LayersManager {
     }
 
     clear() {
+        for (const layer of this.layersMap.values()) {
```

Comment:
`Group.setLayerManager()` calls `LayerManager.removeLayer()` to cleanup, however `LayerManager.removeLayer()` will do nothing if `this.layersMap` doesn't contain the Canvas.